package.json

```diff
@@ -107,8 +55,5 @@
     "*.{ts,js,css,json}": [
       "prettier --write"
```

Comment:
@NSeydoux This is not related to this PR, but shouldn't we run ESLint here as well?

```suggestion
      "eslint --cache --fix",
      "prettier --write"
```